iwyu.cc

```diff
@@ -2548,12 +2548,14 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
 
     const NamedDecl* decl = TypeToDeclAsWritten(type);
 
+    const SourceLocation current_loc = CurrentLoc();
```

Comment:
No need to outline this into a variable, just put the inline `CurrentLoc` calls back